src/reference-manual/statements.qmd

```diff
@@ -630,15 +661,15 @@ $$
 
 Stan allows probability functions to be truncated.  For example, a
 truncated unit normal distributions restricted to $[-0.5, 2.1]$
-can be coded with the following sampling statement.
+can be presented with the following distribution statement.
```

Comment:
I think "coded" is the right verb in this context.  Stan is a programming language!